paywall/src/__tests__/data-iframe/cache/localStorage-no-localStorage.test.js

```diff
@@ -36,6 +37,18 @@ describe('localStorage cache', () => {
       }
     })
 
+    it('merge', async () => {
```

Comment:
Could we try to use better ways to describe tests.
Think that the sentence should __read__ like English, for example: `it('should throw an error when localStorage is not available')`.
This makes it much easier to understand what the test does later on.
Thanks!